### ISSUE TITLE
Sync code between `ubiquity/devpool-directory` and `ubiquity/devpool-directory-private`

### ISSUE THREAD

**Issue by rndquu** [devpool-directory/devpool-directory-tasks#25]
There are 2 repositories which collect issues available for contributors:
1. https://github.com/ubiquity/devpool-directory: Public repo, any contributor can take issue there
2. https://github.com/ubiquity/devpool-directory-private: Private repo, only core contributors can take issues there since that repo issues may contain sensitive/NDA data

The issue is that we somehow need to sync code changes between https://github.com/ubiquity/devpool-directory and https://github.com/ubiquity/devpool-directory-private. We can't simply fork https://github.com/ubiquity/devpool-directory-private from https://github.com/ubiquity/devpool-directory and use github UI since private repos can't be synced.

What should be done:
- when a new code change is introduced in https://github.com/ubiquity/devpool-directory then this code change should also appear in https://github.com/ubiquity/devpool-directory-private

Possible solution described [here](https://github.com/ubiquity/work.ubq.fi/issues/2#issuecomment-2019912818)

**Comment by jordan-ae** [comment]
@rndquu This up for grabs?

**Comment by rndquu** [comment]
> @rndquu This up for grabs?

Yes, you may setup code sync between https://github.com/ubiquity/devpool-directory and your forked instance of https://github.com/rndquu/devpool-directory-private-test for testing purposes and open a PR, I will check how it works with https://github.com/ubiquity/devpool-directory-private then

**Comment by 0x4007** [comment]
I think we should use a git submodule. Basically we set up the private Directory to import all the contents of the public Directory, and then we overwrite the behavior of what needs to be different.

**Comment by rndquu** [comment]
> I think we should use a git submodule. Basically we set up the private Directory to import all the contents of the public Directory, and then we overwrite the behavior of what needs to be different.

Ideally all of the changes should be made in https://github.com/ubiquity/devpool-directory, in particular:
1. Add a new env variable, like `MODE=public|private`
2. If `MODE=private` then disable tweeting
3. Move [opt.json](https://github.com/ubiquity/devpool-directory/blob/6f8273094cee4dd73f2846a3ec3f312d84c71da6/opt.json) and [projects.json](https://github.com/ubiquity/devpool-directory/blob/6f8273094cee4dd73f2846a3ec3f312d84c71da6/projects.json) to env variables
4. Probably smth else

So that we could simply:
1. Fork https://github.com/ubiquity/devpool-directory to https://github.com/ubiquity/devpool-directory-bounties
2. Setup all env variables so that https://github.com/ubiquity/devpool-directory-bounties worked "out of the box" without any code changes
3. Setup code sync as described [here](https://github.com/ubiquity/work.ubq.fi/issues/2#issuecomment-2019912818) so that we could sync code via github UI
